types/ssh2-sftp-client/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for ssh2-sftp-client 5.4
+// Type definitions for ssh2-sftp-client 5.3
```

Comment:
@rxliuli why did you roll the version backwards from .4 to .3? I made changes to this repo a few months ago specifically because we were using the latest version which at the time was .4, and it was not compatible with .3.

I haven't checked but I am worried that by rolling back the version of this repo you'll be making the type declarations incompatible with the version tiself

Reply:
So isn't the latest version now 5.3?  If there is a problem, can you help modify it to the correct version?